xc/common/primitives/ologice3/ologice3.pb_type.xml

```diff
@@ -180,6 +339,12 @@
         </meta>
       </metadata>
     </pb_type>
+
+    <pb_type name="T_INV" blif_model=".subckt T_INV" num_pb="1">
```

Comment:
You probably need a combinatorial delay = "0.0" throught this thing to prevent breaking timing arcs.

Reply:
Fixed. Shouldn't matter anyways as the `T_INV` is only used when routing to const1

Reply:
Sure, but we may end up using T_INV for signals if we teach yosys to emit it